src/main/java/org/hisp/dhis/tasks/DhisAbstractTask.java

```diff
@@ -93,7 +93,7 @@ protected User getUser(Randomizer rnd)
                 return user;
             }
 
-            TestConfig conf = ConfigFactory.create( TestConfig.class );
+            TestConfig conf = org.aeonbits.owner.ConfigFactory.create( TestConfig.class );
```

Comment:
I assume when we refactor this, we would also pass in the config. Or even better only the things that are needed like username/password.

Reply:
That got lost, I can already remove it